### PR TITLE
Increase timeout for assertions

### PR DESCRIPTION
**Pull request by pepoviola**
Prevents timeouts in ci like https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5516019

### CODE REVIEW

File: bridges/testing/tests/0001-asset-transfer/roc-reaches-westend.zndsl [hunk 1]
```diff
@@ -6,7 +6,7 @@ Creds: config
 asset-hub-westend-collator1: run {{ENV_PATH}}/helper.sh with "reserve-transfer-assets-from-asset-hub-rococo-local 5000000000000" within 120 seconds
 
 # check that //Alice received at least 4.8 ROC on Westend AH
-asset-hub-westend-collator1: js-script {{FRAMEWORK_PATH}}/js-helpers/wrapped-assets-balance.js with "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY,4800000000000,Rococo" within 300 seconds
+asset-hub-westend-collator1: js-script {{FRAMEWORK_PATH}}/js-helpers/wrapped-assets-balance.js with "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY,4800000000000,Rococo" within 600 seconds
```
Comment (serban300) [inline bridges/testing/tests/0001-asset-transfer/roc-reaches-westend.zndsl:9]: This should be done also for the other tests: `wnd-reaches-rococo`, etc.

File: bridges/testing/tests/0001-asset-transfer/roc-reaches-westend.zndsl [hunk 2]
```diff
@@ -6,7 +6,7 @@ Creds: config
...
 
 # check that the relayer //Charlie is rewarded by Westend AH
-bridge-hub-westend-collator1: js-script {{FRAMEWORK_PATH}}/js-helpers/relayer-rewards.js with "5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y,0x00000002,0x6268726F,ThisChain,0" within 30 seconds
+bridge-hub-westend-collator1: js-script {{FRAMEWORK_PATH}}/js-helpers/relayer-rewards.js with "5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y,0x00000002,0x6268726F,ThisChain,0" within 60 seconds
```
Comment (serban300) [inline bridges/testing/tests/0001-asset-transfer/roc-reaches-westend.zndsl:12]: This won't help. This is actually ready before the previous check. We can keep it to 30s, or anyway it doesn't matter.

**Review by serban300 — COMMENTED** [review]
Thank you ! Just left 2 small comments.

File: bridges/testing/tests/0001-asset-transfer/roc-reaches-westend.zndsl (hunk 2 above)
Comment (pepoviola) [reply bridges/testing/tests/0001-asset-transfer/roc-reaches-westend.zndsl:12]: Ahh oks, sounds good. So, is the first timeout the issue 👍

**Review by ggwpez — APPROVED** [review]

**Review by serban300 — APPROVED** [review]
Looks good ! If you could also increase the timeout from 300 to 600 for `wroc-reaches-rococo` and `wwnd-reaches-westend` would be great. But it's not a blocker.

**Pull request merged**